src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -22,35 +14,54 @@
 from rucio.client import Client
 from rucio.common.exception import RSEProtocolNotSupported, Duplicate, RSEAttributeNotFound
 
-from DIRAC import S_OK, S_ERROR, gConfig
+from DIRAC import S_OK, S_ERROR, gLogger, gConfig
+from DIRAC.Core.Security import Locations
 from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
-from DIRAC.ConfigurationSystem.Client.Helpers.Registry import (getUserOption, getAllUsers, getHosts,
-                                                               getHostOption, getAllGroups, getDNsInGroup)
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import (getUserOption, getAllUsers, getHosts, getVOs,
+                                                               getHostOption, getAllGroups, getDNsInGroup,
+                                                               getUsersInVO, getGroupsForVO)
 from DIRAC.Core.Base.AgentModule import AgentModule
 from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers, resolveSEGroup
 from DIRAC.Resources.Storage.StorageElement import StorageElement
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getCatalogPath
 
 
 __RCSID__ = "Id$"
 
 
-def getStorageElements():
+def getStorageElements(vo):
   """
   Get configuration of storage elements
 
+  :param vo: VO name that an SE supports
   :return: S_OK/S_ERROR, Value dictionary with key SE and value protocol list
   """
+  log = gLogger.getSubLogger('RucioSynchronizer')
```

Comment:
Maybe it's a good place for using the recently introduced local subLogger https://github.com/DIRACGrid/DIRAC/pull/5284

Reply:
I've modified the logging, please have a look if it is what you wanted. I'll introduce the local logger in a separate commit.

Reply:
I meant to do
```suggestion
  log = gLogger.getLocalSubLogger('RucioSynchronizer/%s' % vo)
```

Reply:
Is the local sub logger well suited for the whole class or only for a method(s) in the module ? Confused here.

Reply:
Only for a method in the module.